source/geometry_model/ellipsoidal_chunk.cc

```diff
@@ -53,11 +53,8 @@ namespace aspect
       semi_major_axis_a (-1),
       eccentricity (-1),
       semi_minor_axis_b (-1),
-      rot_para_to_para_angle (0),
-      para_to_rect_angle (0),
-      rotation_longitude (-1),
-      rotation_latitude (-1),
-      bottom_depth (-1)
+      bottom_depth (-1),
+      corners({{0}})

```

Comment:
clang gives a warning in this line (too many braces for scalar initializer), I think you might as well use the default constructor corners(). If anybody tries to access non-existent entries they will get an error anyway.